metrics/metrics.go

```diff
@@ -48,9 +48,29 @@ var (
 		},
 	)
 
+	DeschedulerLoopDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      DeschedulerSubsystem,
+			Name:           "descheduler_loop_duration_seconds",
+			Help:           "Time taken to complete one full loop of descheduling operation",
+			StabilityLevel: metrics.ALPHA,
+			Buckets:        []float64{0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25},
```

Comment:
is it fair to say that a loop for large clusters can certainly take longer than 25 seconds?
```suggestion
			Buckets:        []float64{0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25, 50, 120},
```

Reply:
I think that would be a fair thing to say based on my previous experience. Will update this accordingly. Should I add a bucket for 250, 500 too ?